packages/monosize/src/commands/measure.mts

```diff
@@ -15,10 +15,11 @@ import type { BuildResult } from '../types.mjs';
 export type MeasureOptions = CliOptions & {
   debug: boolean;
   'artifacts-location': string;
+  fixtures?: string;
```

Comment:
to follow the interface pattern lets make it required and define `*.fixture.js` as default value for the command